optimade/server/routers/index_info.py

```diff
@@ -15,7 +16,7 @@
 
 from optimade.server.config import CONFIG
 
-from .utils import meta_values
+from .utils import meta_values, get_base_url
```

Comment:
```suggestion
from optimade.server.routers.utils import meta_values, get_base_url
```

Reply:
Still waiting for someone to do fix #298 everywhere, but I suppose you're right ;)

Reply:
This change should at least be implemented in this PR for all the lines you touch ;)